src/components/Sections/Restaurants/Restaurants.tsx

```diff
@@ -0,0 +1,17 @@
+import { CustomCardsSection } from "@/components";
+import { RestaurantsData } from "@/data";
+import "./Restaurants.scss";
+/*
```

Comment:
The comments provided are not necessary for such a simple component. Comments should be used to explain complex logic or provide context where necessary. In this case, the code is straightforward and self-explanatory, so the comments can be omitted.